apps/judicial-system/web/src/shared-components/FormFooter/FormFooter.tsx

```diff
@@ -43,6 +44,27 @@ const FormFooter: React.FC<Props> = (props: Props) => {
           {props.nextButtonText ?? 'Halda áfram'}
         </Button>
       )}
+      {props.infoBoxText && (
+        <Box
```

Comment:
Island-ui has a component for this, right? Why not use that?

Reply:
It was only with bold text, looked weird. So i just copied what i needed. It's just a small component.